index.html

```diff
@@ -6,51 +6,71 @@
+</head>
+
 <body>
+<!-- The header part of the code begins here -->
     <header>
-    <div class="pure-nav-list">
-            <h4 class="important"><a href="#">Ayodele</a></h4>
-            <nav>
-                <div class="menu1">
-                    <div class="menu"></div>
-                    <div class="menu"></div>
-                    <div class="menu"></div>
-                </div>
+        <div class="flexcontainer-header">
+            <a href="#"> Ayodele </a>
+            <nav class="navigation">
+                <ul>
+                <li><i class="fas fa-bars"></i></li>
+                </ul>
             </nav>
-            </div>
-            
-        </header>
+        </div>
+    </header>
+    <!-- The header part of the code ends here -->
 
-        <main>
-    <section class="section-headline">
-    <div class="main-text Developer">
-    <div class="introduction">
-    <h1>Hey There. <br> I’m Ayodele</h1>
-    <h3>I’am a Software Developer</h3>
-    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-    
+<!-- The content part of the code begins here -->
+<main class="flexcontainer-content">
+    <section class="box-content headline">
+    <h1>Hey There. <br> I'm Ayodele</h1> 
+    <div class="box-content product">
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
     </div>
+    <!-- <p class="box-content skill">I'm a software Developer!</p>
+        <P class="box-content product">I can help you build a product, feature or <br> website. Look through some of
+        my work <br> and Experience! if you like what you see <br> and have a project you need coded, <br> dont
+        hesitate to contact me.
+    </p> -->
+    <!-- Code for social media -->
+    <div class="box-content social-media"><img src="images/img_social.png" alt="icons"></div>
+    <div id="background"></div>
+    </section>
+    <!-- The second section of the code begins here  -->
+    <section class="gym-fit">
+        <div class="works">
+            <h4>My Recent Works</h4>
+        </div>
 
-    <ul class=" social-media social-media-icons mv-social-media mv-social-media-icons">
-        <li class="social-1"><a href="#"><i class="fab fa-github fa-lg"></i></a></li>
-        <li class="social-2"><a href="#"><i class="fab fa-linkedin-in fa-lg"></i></a></li>
-        <li class="social-3"><a href="#"><i class="fab fa-angellist fa-lg"></i></a></li>
-        <li class="social-4"><a href="#"><i class="fab fa-twitter fa-lg"></i></a></li>
-        <li class="social-5"><a href="#"><i class="fab fa-medium-m"></i></a></li>
-        </ul>
+        <hr class="solid"/>
 
+    <div class="pictures"></div>
```

Comment:
- [ ] Hi, well done on laying out this section. However, the requirements state that you should use `grid` to layout the projects' container. right now you are using `grid` on just a single `div` that has no children. Using `grid` on an element with no children is pointless as `grid` and `flex` are supposed to be used on containers in order to layout their children. Make sure to use `grid` on the container of the projects in order to layout the projects themselves
- [ ] You need more projects to be laid out currently you only have one project. Please add more projects. because you will need to be able to display all the projects you've done in your portfolio 👍 
- [ ] For each project that specific project should be laid out using `flexbox`. Make sure to also do this for every project inside the projects' container 👍.